tests/cmd/stability/main.go

```diff
@@ -219,5 +222,19 @@ func main() {
 	// truncate a sst file and check failover
 	oa.TruncateSSTFileThenCheckFailoverOrDie(cluster1, 5*time.Minute)
 
+	// stop one etcd node and k8s/operator/tidbcluster is available
+	faultEtcd := tests.SelectNode(conf.ETCDs)
+	fta.StopETCDOrDie(faultEtcd)
+	defer fta.StartETCDOrDie(faultEtcd)
+	oa.CheckOneEtcdDownOrDie(operatorCfg, allClusters, faultEtcd)
```

Comment:
You should add other cases: stopping 2 etcds and stopping 3 etcds.

Reply:
Sure, I will add these cases in next pr